packages/pull-signal/src/model/domain/errors.ts

```diff
@@ -20,14 +20,14 @@ export function requestValidationError(message: string): ApiError<ErrorCodes> {
 }
 
 export function operationPullForbidden({
-  purposeId,
   eserviceId,
+  consumerId,
 }: {
-  purposeId: string;
+  consumerId: string;
   eserviceId: string;
 }): ApiError<ErrorCodes> {
   return new ApiError({
-    detail: `Insufficient privileges: cannot access to pull signals with voucher's purpose ${purposeId} for e-service PULL; please verify if the consumer has a valid agreement to pull signals to e-service ${eserviceId}`,
+    detail: `Insufficient privileges: cannot access to pull signals with voucher's organization ${consumerId} for e-service PULL. Please verify if the consumer has a valid agreement and a valid purpose to pull signals to e-service ${eserviceId}. Verify also if the e-service is enabled for using Signal Hub.`,
```

Comment:
Should we align error message with push-service message? Should be: "I**nsufficient privileges: producerId: ${producerId} cannot access to pull signal for e-service with id ${eserviceId}; Please verify if you have a valid agreement and a valid purpose to pull signals. Verify also if that e-service is enabled for using Signal Hub.**"

Reply:
Got it, I'try to rewrite in this way:
`Insufficient privileges: your organization (id: ${consumerId}) cannot pull signals for an e-service with id ${eserviceId}. Please verify if you have a valid agreement and at least one valid purpose to pull signals. Verify also if that e-service is enabled for using Signal Hub.`